### ISSUE TITLE
Using a reference for the variable on range scope return

### ISSUE THREAD

**Issue by KevinKlinger** [kyoh86/scopelint#5]
```
for _, i := range f {
	if i.FileNumber == fileID {
		return &i, nil
	}
}
```

This delivers: Using a reference for the variable on range scope `i`

I'm unsure if this is a false positive. From my point of view this should be save, since iterating over f will be stopped as soon as I return.

What's your opinion?

**Comment by vnovikau** [comment]
I faced with the same case; any updates on it? thanks.

**Comment by kyoh86** [comment]
oh I see.
I'll fix it on the weekend.

**Comment by vnovikau** [comment]
@kyoh86 great, thank you!
